data/json/recipes/chem/fuel.json

```diff
@@ -88,5 +88,19 @@
     ],
     "//": "Making kerosene from (tainted) fat directly is less efficient than using lard/tallow, but allows processing of spoilt fats. See #29770.  Needs 10% of volume of the fats in washing soda.  I've reduced the yield percentages from what was acheived in this laboratory based article Performance of Thermochemical Conversion of Fat, Oils, and Grease into Kerosene-like Fuel in Different Production Scales.",
     "flags": [ "ALLOW_ROTTEN" ]
+  },
+  {
+    "result": "crude_lamp_oil",
+    "type": "recipe",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_FUEL",
+    "skill_used": "chemistry",
+    "difficulty": 2,
+    "time": "5 s",
```

Comment:
So the recipe is just to mix some fat without any kind of transformation and it works?

Reply:
Yes, as far as I understood the sources that i cited above correctly, vegetable oil or animal fat was just used as is. So theoretically, if it would be possible to use these items as ammo directly, that would certainly be more realistic, but also more complicated.